src/components/common/roomOptions/Picker.js

```diff
@@ -25,14 +26,40 @@ class PickerColumn extends Component {
     };
   }
 
-  // eslint-disable-next-line react/no-deprecated
-  componentWillReceiveProps(nextProps) {
+  UNSAFE_componentWillReceiveProps(nextProps) {
     if (this.state.isMoving) {
       return;
     }
     this.setState(this.computeTranslate(nextProps));
   }
 
+  componentDidMount() {
+    document.addEventListener("keydown", this.handleKeyDown);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener("keydown", this.handleKeyDown);
+  }
+
+  handleKeyDown = (event) => {
```

Comment:
```suggestion
  handleKeyDown(event) {
```
클래스형에서는 이렇게 해야 하지 않나요?